src/android/InAppBrowserDialog.java

```diff
@@ -49,8 +49,8 @@ public void onBackPressed () {
             // because it does a clean up
             if (this.inAppBrowser.hardwareBack() && this.inAppBrowser.canGoBack()) {
                 this.inAppBrowser.goBack();
-            }  else {
-                this.inAppBrowser.closeDialog();
```

Comment:
Is it true that we *never* wanted to close the dialog? Or are there some circumstances where we may want to close it? And why?

Reply:
While we're at it, can we change `Broswer` to `Browser` above?

Reply:
So in discussion with Steve, we do close the dialog when we log out, which is done in the `InAppBrowser` class - that does a `document.reload` which calls an `InAppBrowser.onReset`